aioftp/server.py

```diff
@@ -1511,12 +1586,18 @@ def get_paths(self, connection, path):
 
         if not connection.future.passive_server.done():
 
-            connection.passive_server = await asyncio.start_server(
-                handler,
-                connection.server_host,
-                0,
-                loop=connection.loop,
-            )
+            coro = self._start_passive_server(connection, handler)
+
+            try:
+
+                connection.passive_server = await coro
+
+            except errors.NoAvailablePort:
+
+                connection.response("421", ["no free ports"])
+
+                return False
```

Comment:
as per rfc we should do this:

>421 Service not available, closing control connection.

and there're no other possible replies to PASV except 227, 421 and few 5xx